apps/full-stack-tests/src/Utils.ts

```diff
@@ -34,3 +51,49 @@ export function safeDispose(disposable: {} | { [Symbol.dispose]: () => void } |
     disposable[Symbol.dispose]();
   }
 }
+
+type GetNodesRequestOptions = HierarchyRequestOptions<IModelConnection, NodeKey, RulesetVariable> & ClientDiagnosticsAttribute;
+type GetContentRequestOptions = ContentRequestOptions<IModelConnection, Descriptor | DescriptorOverrides, KeySet, RulesetVariable> & ClientDiagnosticsAttribute;
+type GetDistinctValuesRequestOptions = DistinctValuesRequestOptions<IModelConnection, Descriptor | DescriptorOverrides, KeySet, RulesetVariable> &
+  ClientDiagnosticsAttribute;
+
+type MultipleValuesRequestOptions = Paged<{
+  /**
+   * Max number of requests that should be made to the backend to fulfill the whole request.
+   * `undefined` means no limit, so in that case all requests are sent at once.
+   */
+  maxParallelRequests?: number;
+  /**
+   * Size of a single batch when fetching data through multiple requests. If not set,
+   * the fall back is requested page size. If the page size is not set, the backend
+   * decides how many items to return.
+   */
+  batchSize?: number;
+}>;
+
+type WithIterableMethods<T extends PresentationManager> = {
+  getNodesIterator(requestOptions: GetNodesRequestOptions & MultipleValuesRequestOptions): Promise<{
+    total: number;
+    items: AsyncIterableIterator<Node>;
+  }>;
+  getContentIterator(requestOptions: GetContentRequestOptions & MultipleValuesRequestOptions): Promise<
+    | {
+        descriptor: Descriptor;
+        total: number;
+        items: AsyncIterableIterator<Item>;
+      }
+    | undefined
+  >;
+  getDistinctValuesIterator(requestOptions: GetDistinctValuesRequestOptions & MultipleValuesRequestOptions): Promise<{
+    total: number;
+    items: AsyncIterableIterator<DisplayValueGroup>;
+  }>;
+} & T;
+
+export function isIterableManager(manager: PresentationManager): manager is WithIterableMethods<PresentationManager> {
+  return "getNodesIterator" in manager && "getContentIterator" in manager && "getDistinctValuesIterator" in manager;
+}
+
+export function isSelectionStorageSupported() {
+  return !frontendPackageJson.version.startsWith("4.4.");
```

Comment:
I think the following is easier to understand and won't break if we change the core version in regression pipeline from 4.4 to something else.

```suggestion
  // selection storage support was added in 4.5
  const [major, minor] = frontendPackageJson.version.split(".").map(Number.parseInt);
  return major > 4 || major === 4 && minor >= 5;
```